x/auth/keeper/keeper.go

```diff
@@ -130,6 +131,7 @@ func NewAccountKeeper(
 	ak := AccountKeeper{
 		addressCodec:          ac,
 		validatorAddressCodec: vac,
+		consensusAddressCodec: cac,
```

Comment:
I actually think we do not need this accessible from auth. I've made for consistency, but user from depinject can simply use app config and get the consensus address codec from everywhere if wish to.